tools/base/fpgad/plugins/fpgad-vc/fpgad-vc.c

```diff
@@ -228,9 +228,18 @@ STATIC fpga_result vc_sensor_get_u64(vc_sensor *sensor,
 	return res;
 }
 
+// The percentage by which we adjust the power trip
+// points so that we catch anomolies before the hw does.
+#define VC_PERCENT_ADJUST_PWR 5
```

Comment:
Why not express this as 0.05?
Or treat is as the not how much we subtract but as a percentage of the value?
```C
#define VC_ADJUSTED_PERCENT_PWR 0.95

//...

s->high_fatal *= VC_ADJUSTED_PERCENT_PWR;

```

Reply:
Both the values and the trip points are uint64_t's.  I just didn't want to go the floating point route.